src/main/zapHomeFiles/hud/drawer.html

```diff
@@ -20,24 +21,50 @@
     </div>
 
     <template id="history-template">
-        <table class="table table-striped table-hover table-scroll table-history" style="min-width: fit-content;">
-            <thead>
-                <tr>
-                    <th style="width: 125px"> {{ $t('message.history_http_message_field_time') }} </th>
-                    <th style="width: 75px"> {{ $t('message.history_http_message_field_code') }} </th>
-                    <th style="width: 75px"> {{ $t('message.history_http_message_field_method') }} </th>
-                    <th style="width: 800px"> {{ $t('message.history_http_message_field_url') }} </th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr v-for="message in messages" @click="messageSelected(message.id)" :id="'message-tr-'+message.id">
-                    <td> {{ message.time }} </td>
-                    <td> {{ message.code }} </td>
-                    <td> {{ message.method }} </td>
-                    <td> {{ message.url }} </td>
-                </tr>
-            </tbody>
-        </table>
+        <div>
+            <table id="history-header" class="table">
+                <thead>
+                    <tr>
+                        <th class="title time"> 
+                            {{ $t('message.history_http_message_field_time') }}
+                        </th>
+                        <th class="title code"> 
+                            {{ $t('message.history_http_message_field_code') }}
+                        </th>
+                        <th class="title method"> 
+                            {{ $t('message.history_http_message_field_method') }}
+                        </th>
+                        <th class="title url"> 
+                            {{ $t('message.history_http_message_field_url') }}
+                        </th>
+                        <th class="title filter"> 
+                            <span>
+                                <input id="history-filter" v-model="filter" type="text" v-bind:class="{ 'error': isRegExError }" placeholder="Filter" />
+                            </span>
+                            <span>
+                                <input v-model="regexEnabled" type="checkbox" />
+                            </span>
+                            <span>
+                                {{ enableRegExText }}
+                            </span>
+                        </th>
+                        <th class="title hidden">
+                            <span>{{ hiddenMessageCount }} {{ commonOf }} {{ messageCount }} {{ historyItemsFilteredSuffix }}</span>
```

Comment:
Just noticed this - we're assuming the text structure is "x of y etc" which is obviously right in English but might not be right in other languages, eg Japanese / Chinese etc? 
I think the solution is to have a parameterized i18n string like: 
history_of = {0} of {1} items hidden by filters
and to pass the values as params

Reply:
Good catch! I'll take care of that today 👍

Reply:
@psiinon Updated w/ latest request!